test/models/book_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
Do you need this?

Reply:
no, when rsepc didnt work as i wanted y tried that out, i can remove it now

Reply:
Please, we are gonna work with rspec.